### ISSUE TITLE
More/better customisation of the "active/highlighted/current" line in synced lyrics

### ISSUE THREAD

**Issue by jacquesh** [jacquesh/foo_openlyrics#30]
Requested here: https://www.reddit.com/r/foobar2000/comments/mm70rb/i_made_an_opensource_alternative_to/h58wwjy/

In particular allowing you to make the current line have a larger font size

**Comment by jacquesh** [comment]
Note to my future self: I tried this because it seemed really simple, and it actually would be if not for the fact that both position and colour of the highlighted line gets interpolated smoothly when it goes in and out of being the active line. While we can scroll smoothly and interpolate colours smoothly, it appears that this doesn't work nearly as well for font sizes.

The attached video has an example of the stuttery resizing behaviour I'm getting with what I tried (it actually looked like this, its not stuttery because of low fps on the video):
https://user-images.githubusercontent.com/4639662/153944964-986756c7-b717-4f2b-94f4-82246c8b1381.mp4

On top of this, it looks really bad how the text gets re-flowed as it grows/shrinks. Maybe I'll come back to this later when I'm looking to spend more time on it.
